pkgs/db/prisma/migrations/20230623153054_/migration.sql

```diff
@@ -17,7 +17,7 @@ INSERT INTO "public"."Users"(
   "updatedAt")
 VALUES (
   'githubapp',
-  'Github App',
+  'GitHub App',
   '/github-mark.png',
   'support+githubapp@specfy.io',
   NOW(),
```

Comment:
Mmh this migration was already applied in prod, I wonder what will happen. 
You can leave it like this but I need to double check before deploying